Source/Core/DolphinQt/Settings.cpp

```diff
@@ -389,6 +387,26 @@ void Settings::ApplyStyle()
     stylesheet_contents.append(QStringLiteral("%1").arg(tooltip_stylesheet));
   }
 
+  // For Fusion, define group box style if not already defined.
+  if (style_name.compare(QStringLiteral("fusion"), Qt::CaseInsensitive) == 0 &&
+      !stylesheet_contents.contains(QStringLiteral("QGroupBox")))
+  {
+    stylesheet_contents.append(QStringLiteral("QGroupBox { "
+                                              "    margin-top: 0.6em; "
+                                              "    padding-top: 0.5em; "
+                                              "    padding-bottom: 0; "
+                                              "    padding-left: 1px; "
```

Comment:
I'm probably being picky, but that's a lot of whitespace in these literals that no one will ever see.

Reply:
How do you find this:

```cpp
    stylesheet_contents.append(
        QStringLiteral("QGroupBox { margin-top: 0.6em; padding-top: 0.5em; padding-bottom: 0; "
                       "padding-left: 1px; padding-right: 1px; } "
                       "QGroupBox::title { subcontrol-origin: margin; subcontrol-position: top "
                       "left; left: 0.7em; padding-top: 1px; min-width: 0; }"));
```

(Suggestions welcome; I dislike both equally.)

Reply:
That's fine, or maybe moving the one space to the start of each line rather than the end of the line?

Reply:
How about:

```cpp
    stylesheet_contents.append(QStringLiteral(R"(
      QGroupBox {
        margin-top: 0.6em;
        padding-top: 0.5em;
        padding-bottom: 0;
        padding-left: 1px;
        padding-right: 1px;
      }
      QGroupBox::title {
        subcontrol-origin: margin;
        subcontrol-position: top left;
        left: 0.7em;
        padding-top: 1px;
        min-width: 0;
    })"));
```

Reply:
That's going to add a heck ton of whitespace for each line.

Something like this is what I have in mind, but again, I'm being picky, and others can share their opinions :)
```cpp
"QGroupBox {"
" margin-top: 0.6em;"
" padding-top: 0.5em;"
" padding-bottom: 0;"
" padding-left: 1px;"
" padding-right: 1px;"
"} "
"QGroupBox::title {"
" subcontrol-origin: margin;"
" subcontrol-position: top left;"
" left: 0.7em;"
" padding-top: 1px;"
" min-width: 0;"
"}"
```

Reply:
Updated with your suggestion.